src/config/mangata-dev.js

```diff
@@ -0,0 +1,200 @@
+const assets = [
```

Comment:
This config needs to be updated later.

These are the assets I grabbed from Mangata, but their dev environment has fewer parameters.